### PR TITLE
feat(app): Renamed token to tokenName in liquidity flow modals

### PR DESCRIPTION
**Pull request by Artem1211**

### CODE REVIEW

File: packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export function BorrowDecreaseModal({
         title="How much to pay off"
         buttonProps={{
           label: `Pay off ${value} ${tokenSymbol}`,
-          onClick: () => onSend({ value: BigInt(value), token }),
+          onClick: () => onSend({ value: BigInt(value), token: tokenName }),
```
Comment (bouvens) [inline packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx:67]: It seems we can update `onSent` to take `tokenName` as well.

File: packages/app/src/features/liquidity-flow/hooks/use-lend-decrease.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const useLendDecrease = (): {
         deposit={deposit.value}
         debtSumUsd={debtSumUsd}
         depositSumUsd={depositSumUsd}
-        token={deposit.token}
+        tokenName={deposit.token}
```
Comment (bouvens) [inline packages/app/src/features/liquidity-flow/hooks/use-lend-decrease.tsx:64]: Can we update `token` property in position to `tokenName`?

File: packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx (hunk 1 above)
Comment (Artem1211) [reply packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx]: Updated

Comment (bouvens) [reply packages/app/src/features/liquidity-flow/components/borrow-decrease-modal/borrow-decrease-modal.tsx]: Thank you.

**Review by bouvens — APPROVED** [review]

**Pull request merged**